scripts/pipenv_run_tests.sh

```diff
@@ -1,2 +1,2 @@
 #! /bin/bash
-docker exec -i faf-db mysql faf < test-data.sql && scripts/run_tests_with_coverage.sh $@
+docker exec -i faf-db mysql faf < test-data.sql && scripts/run_tests_with_coverage.sh "$@"
```

Comment:
Are you sure this will still work? The point is to forward all command line arguments to pytest so that for example you can do `pipenv run tests tests/integration_tests -s --cli-log-level 5`. I would think that adding quotes would mash all of your arguments into a single string, which presumably doesn't get parsed correctly.

Reply:
Here's the warning from ShellCheck that I get when I leave it in its current state:

"Double quote array expansions to avoid re-splitting elements."

> Rationale:
> Double quotes around $@ (and similarly, ${array[@]}) prevents globbing and word splitting of individual elements, while still expanding to multiple separate arguments.
> 
> Let's say you have three arguments: baz, foo bar and *
> 
> "$@" will expand into exactly that: baz, foo bar and *
> 
> $@ will expand into multiple other arguments: baz, foo, bar, file.txt and otherfile.jpg
> 
> Since the latter is rarely expected or desired, ShellCheck warns about it.

https://github.com/koalaman/shellcheck/wiki/SC2068

Reply:
I just tried it on my machine, and it seems to still work so I guess there's no problem.